sdk/opendp/whitenoise/synthesizers/pytorch/nn/__init__.py

```diff
@@ -8,7 +8,7 @@
 import torch.optim as optim
 from torch.utils.data import DataLoader, TensorDataset
 
-from torchdp import PrivacyEngine, utils, autograd_grad_sample
+from opacus import PrivacyEngine, utils, autograd_grad_sample
```

Comment:
we should update the requirements.txt that need opacus to replace the pytorch-dp dep with opacus. 

I suspect that will fix the builds

Reply:
I added the change with github ui :)